boards/Makefile.common

```diff
@@ -77,6 +81,9 @@ check_defined = $(strip $(foreach 1,$1,$(if $(value $1),,$(error Undefined varia
 $(call check_defined, PLATFORM)
 $(call check_defined, TARGET)
 
+# Location of target-specific build
+TARGET_PATH = $(TARGET_DIRECTORY)$(TARGET)
```

Comment:
Do we need this?

Reply:
it is a common sub expression elimination within makefile (used later for location of target binaries), and it also defines a useful variable we use in board-specific script (code signing, flashing, etc). I found it more convenient than hard-coded  structure which may change over time.